sdk/devcenter/azure-developer-devcenter/README.md

```diff
@@ -8,121 +8,230 @@ Use the package for Azure DevCenter to:
-[azure-identity][azure_identity_credentials] library.
+### Create Client with an Azure Active Directory Credential
 
-To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip]
+In order to interact with the Dev Center service, you will need to create an instance of a client. An **endpoint** and **credential** are necessary to instantiate the client object.
+
+For the endpoint use the Dev Center URI. It should have the format `https://{tenantId}-{devCenterName}.{devCenterRegion}.devcenter.azure.com`.  Set this value as the environment variable for `DEVCENTER_ENDPOINT`.
+
+For the credential use an [Azure Active Directory (AAD) token credential][authenticate_with_token], providing an instance of the desired credential type obtained from the [azure-identity][azure_identity_credentials] library.
+
+To authenticate with AAD, you must first install [azure-identity][azure_identity_pip] using [pip][pip]
+
+```bash
+pip install azure-identity
+```
 
 After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
-As an example, [DefaultAzureCredential][default_azure_credential] can be used to authenticate the client:
 
+As an example, [DefaultAzureCredential][default_azure_credential] can be used to authenticate the client:
 Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
 `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`
 
-Use the returned token credential to authenticate the client:
+Use the returned token credential to authenticate the client.
+
+<!-- SNIPPET:create_devbox_sample.create_dev_center_client -->
 
 ```python
->>> import os
->>> from azure.developer.devcenter import DevCenterClient
->>> from azure.identity import DefaultAzureCredential
->>> tenant_id = os.environ['AZURE_TENANT_ID']
->>> endpoint = os.environ["DEVCENTER_ENDPOINT"]
->>> client = DevCenterClient(endpoint, credential=DefaultAzureCredential())
+import os
+
+from azure.developer.devcenter import DevCenterClient
+from azure.identity import DefaultAzureCredential
+
+# Set the values of the dev center endpoint, client ID, and client secret of the AAD application as environment variables:
+# DEVCENTER_ENDPOINT, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET
+try:
+    endpoint = os.environ["DEVCENTER_ENDPOINT"]
+except KeyError:
+    raise ValueError("Missing environment variable 'DEVCENTER_ENDPOINT' - please set it before running the example")
+
+# Build a client through AAD
+client = DevCenterClient(endpoint, credential=DefaultAzureCredential())
 ```
 
+<!-- END SNIPPET -->
+
+With `DevCenterClient` you can manipulate operations in [Dev Center, Dev Box and Environments REST operations group](https://learn.microsoft.com/rest/api/devcenter/developer/operation-groups).
+
 ## Examples
+* [Create, Connect and Delete a Dev Box](#menage-dev-boxes)
+* [Deploy and Delete Environments](#menage-environments)
```

Comment:
These are causing your build errors, there's a misspelling also make sure it matches the sections you're referencing